### DISCUSSION TITLE
Question About ProjectRed

### DISCUSSION THREAD

**Discussion by Flexico** [cc-tweaked/CC-Tweaked#931] [Q&A] [answered]
So Project: Red is out for MC 1.16.5! Which side is responsible for any future integration with the bundled cables? In other words, where should I go to check up on any progress in this department (if any is planned)?

**Comment by SquidDev** [comment, marked as answer]
This is typically handled by ProjectRed - they have a [separate integration mod](https://www.curseforge.com/minecraft/mc-mods/project-red-integration). It looks like the latest beta supports CC - see https://github.com/MrTJP/ProjectRed/commit/e222759924530a83c808642384bf525eb319652d.

**Reply by Flexico** [reply]
I have that mod installed already! The bundled cables don't connect to computers though. :(

**Reply by Flexico** [reply]
Oh wait, you sad latest BETA! I haven't tried that yet! *scampers off to download and install*
